src/Views/batchprocessor.html

```diff
@@ -15,7 +15,32 @@ <h4 class="modal-title">StreamStats Batch Processor</h4>
     <div style="padding-top: 10px;">
 
         <div id="submitBatch" ng-if="vm.selectedBatchProcessorTabName==='submitBatch'">
-            <p>This is the submit batch processor tab</p>
+            <div id="selectRegionID">
+                <label>Select Region / State</label>
+                <select ng-model="selectedRegion" ng-options="x.name for x in vm.regionList | orderBy: 'name'"></select>
+            </div>
+            <div>
+                <span>
+                    <label class="chx">
+                        <input id="cbDelineate", type="checkbox" ng-checked="selectedRegion">

```

Comment:
Once the user selects a State/Region, the user should not be allowed to uncheck this box

Reply:
fixed with ```onclick="return false;"